src/main/java/uk/gov/pay/adminusers/model/Service.java

```diff
@@ -31,6 +34,15 @@
     private boolean collectBillingAddress;
     private GoLiveStage goLiveStage;
     private boolean experimentalFeaturesEnabled;
+    private String sector;
```

Comment:
What values can `sector` have? Might it be a good candidate for an enum?

Reply:
Thoughts were that because it's only for reporting we'll only constrain this in toolbox.
Makes it easier to add to or change when we need to

Reply:
Plan is to have a drop-down in toolbox of allowed values